magma/lib/magma/flag.rb

```diff
@@ -0,0 +1,17 @@
+class Magma
+  class Flag < Sequel::Model
+  end
+
+  module Flags
+    # All modules nested under this namespace represent flags that can be used in a production.
+    # Each module should define a NAME constant representing the name of a flag and each
+    # module should also specify all potential values that can be assigned to a flag.
+    module GnomonMode
```

Comment:
Not sure if we like this approach, but I think it is important to have a record somewhere of all the possible values a flag can have. This could also be stored in the db.

Reply:
I think i like it in code as we cannot just create modes on the fly by addung db entries so the db doesn't really make sense here. Might be easier as a Hash instead of a nested Module?